### PR TITLE
handle passing in None for a channels route

### PR DESCRIPTION
**Pull request by tmbo**
**Proposed changes**:
- properly handle if `route` is set to `None` for a blueprint of an input channel when registering

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [ ] updated the changelog

### CODE REVIEW

File: tests/test_channels.py [hunk 1]
```diff
@@ -747,3 +751,26 @@ def test_newsline_strip():
...
+    """Check we properly connect the input channel blueprint if route is None"""
+    from rasa_core.channels import RestInput
+
+    # load your trained agent
+    agent = Agent.load(MODEL_PATH, interpreter=RegexInterpreter())
+
+    input_channel = RestInput()
+
+    from flask import Flask
+    import rasa_core
```
Comment (MetcalfeTom) [inline tests/test_channels.py]: Is this entire import necessary?

Comment (tmbo) [reply tests/test_channels.py]: yes, for `rasa_core.channels.channel.register`

Comment (MetcalfeTom) [reply tests/test_channels.py]: I was able to test this one locally by importing `channel` and then using `channel.register`

Comment (tmbo) [reply tests/test_channels.py]: yes sure, but that doesn't make any difference (ok it makes the line a little shorter where we call `register` but thats it :D - given that this is the way we call it in the other channels as well I don't see a reason to change this)

**Pull request merged**